pytorch_lightning/utilities/cloud_io.py

```diff
@@ -25,9 +25,15 @@
 
 
 def load(path_or_url: str, map_location=None):
-    if urlparse(path_or_url).scheme == "" or Path(path_or_url).drive:  # no scheme or with a drive letter
-        return torch.load(path_or_url, map_location=map_location)
-    return torch.hub.load_state_dict_from_url(path_or_url, map_location=map_location)
+    try:
+        fs = get_filesystem(path_or_url)
+    except ValueError:
```

Comment:
exception is thrown here: https://github.com/intake/filesystem_spec/blob/master/fsspec/registry.py#L190-L191